app/forms/hyrax/forms/collection_form.rb

```diff
@@ -79,6 +79,14 @@ def thumbnail_title
         model.thumbnail.title.first
       end
 
+      def available_child_collections
+        load_child_collections
```

Comment:
Why do we need this method that just calls another method?

Reply:
We probably don't.